internal/app/machined/pkg/controllers/network/link_spec.go

```diff
@@ -219,6 +219,16 @@ func (ctrl *LinkSpecController) syncLink(ctx context.Context, r controller.Runti
 	case resource.PhaseRunning:
 		existing := findLink(*links, link.TypedSpec().Name)
 
+		if existing != nil && existing.Attributes.Info == nil {
```

Comment:
would `existing.Attributes` ever be nil?

Reply:
nevermind, saw the comment in the other file

Reply:
That's a good question: `findLink` actually looks into `link.Attributes.Name` and there is no nil pointer panic there in #6956. But `LinkMessage.UnmarshalBinary` doesn't make any promises :)